sharding/nodesCoordinator/indexHashedNodesCoordinator_test.go

```diff
@@ -129,7 +129,7 @@ func createArguments() ArgNodesCoordinator {
 			IsRefactorPeersMiniBlocksFlagEnabledField: true,
 		},
 		ValidatorInfoCacher:             &vic.ValidatorInfoCacherStub{},
-		StakingV4EnableEpoch:            stakingV4Epoch,
+		StakingV4Step2EnableEpoch:       stakingV4Epoch,
```

Comment:
typo in `strikeV4Epoch` ? :D
Also misalligned value and extra space before `:`

Reply:
There is no typo there or anywhere in the project;
Also, formated the code again, there is no misalligned.
Perhaps you were reviewing an older commit?

Reply:
yup, probably.